examples/veRL/multi_env/config/grpo_trainer.yaml

```diff
@@ -1,34 +1,44 @@
+reasoning_gym:
```

Comment:
Would recommend we add the defaults section here to minimise friction when updating `verl` versions in future, see their examples: https://github.com/volcengine/verl/blob/main/examples/sglang_multiturn/config/gsm8k_multiturn_grpo.yaml